bigbone/src/main/kotlin/social/bigbone/api/entity/OEmbedMetadata.kt

```diff
@@ -0,0 +1,84 @@
+package social.bigbone.api.entity
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+import social.bigbone.api.method.OEmbedMethods
+
+/**
+ * oEmbed metadata returned by [OEmbedMethods].
+ * @see <a href="https://oembed.com/#section2.3">OEmbed response parameters documentation</a>
+ */
+@Serializable
+data class OEmbedMetadata(
+    /**
+     * The resource type.
+     * It’s unclear from <a href="https://docs.joinmastodon.org/methods/oembed/#get">Mastodon’s documentation</a>
```

Comment:
Looking at Mastodon's OEmbed Serializer, I can see that **rich** is the only value they deliver at the moment: https://github.com/mastodon/mastodon/blob/main/app/serializers/oembed_serializer.rb.

Reply:
Ah, thanks for checking! I’ll update the documentation and default value.